module/importer-ergonode-1/src/Infrastructure/Processor/Step/ErgonodeAttributesProcessorStep.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Ergonode\ImporterErgonode\Infrastructure\Processor\Step;
+
+use Ergonode\Attribute\Domain\ValueObject\AttributeCode;
+use Ergonode\Attribute\Domain\ValueObject\AttributeScope;
+use Ergonode\EventSourcing\Infrastructure\Bus\CommandBusInterface;
+use Ergonode\Importer\Domain\Entity\Import;
+use Ergonode\ImporterErgonode\Domain\Command\Import\ImportAttributeCommand;
+use Ergonode\ImporterErgonode\Infrastructure\Processor\ErgonodeProcessorStepInterface;
+use Ergonode\ImporterErgonode\Infrastructure\Reader\ErgonodeAttributeReader;
+use Ergonode\SharedKernel\Domain\Aggregate\AttributeId;
+
+class ErgonodeAttributesProcessorStep implements ErgonodeProcessorStepInterface
+{
+    private const FILENAME = 'attributes.csv';
```

Comment:
Shouldn't this be injected? Different places are now assuming they are using right naming

Reply:
Nope. This will always be static. Export files names are the same every time

Reply:
I know it's static but currently, truth is that two completely different classes with no correlation whatsoever are using "mysterious" filename. Easily could happen that someone would change in one place but not in the other. Or am I wrong?

Reply:
Those filename should not change in export ZIP file. It should be able to add new, but changing them should be prohibited